src/pages/requestAccess/requestAccessView.js

```diff
@@ -140,14 +137,25 @@ function requestAccessView({ data, classes }) {
     }
   };
 
+  const validateFields = () => {
+    const fieldsToChk = ['firstName', 'lastName', 'organization'];
```

Comment:
This Fields should be read from custodian configuration file. Only required fields should be checked.

Reply:
Change has been implemented